src/run.js

```diff
@@ -349,6 +353,9 @@ const processPage = ({
         // of all the ones we're going to assess. For style elements,
         // also extract each tag's content.
         Array.from(document.querySelectorAll('link, style')).forEach(elem => {
+          if (isResourceHinted(elem)) {
```

Comment:
you can update `isStylesheetLink` instead.

```js
const isStylesheetLink = elem =>
          elem.tagName === 'LINK' &&
          elem.href &&
          (elem.rel.toLowerCase() === 'stylesheet') &&
          !(
            elem.href.toLowerCase().startsWith('data:') ||
            elem.href.toLowerCase().startsWith('blob:') ||
            elem.media.toLowerCase() === 'print'
          );
```

Reply:
yup, done.